src/components/Calculator.js

```diff
@@ -1,63 +1,50 @@
 /* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable all ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust  and clean code.

Reply:
Thank you! :+1: